packages/semantic-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -12,19 +12,19 @@ import { getSemanticProps } from "../util";
 
 function selectValue<S extends StrictRJSFSchema = RJSFSchema>(
   value: EnumOptionsType<S>["value"],
-  selected: any,
+  selected: any[],
   all: any[]
```

Comment:
Can we use these more accurate types for the other declarations of this function (or if moving to utils, use these types there)? Same for `deselectValue`.

Reply:
Yeah, I'll pull this one into utils